src/MongoDB.AspNetCore.OData/MongoEnableQueryAttribute.cs

```diff
@@ -73,6 +81,12 @@ public override IQueryable ApplyQuery(IQueryable queryable, ODataQueryOptions qu
         return queryable;
     }
 
+    private Expression GetRewrittenExpression(IQueryable queryable)
+    {
+        // Use MongoExpressionRewriter to update the substring methods
```

Comment:
We should not mention substring here, because we might find more non-translatable expressions in future.
`// Use MongoExpressionRewriter to rewrite non-translatable methods if needed.`

Reply:
Done.